lib/src/dom/top_level_queries.dart

```diff
@@ -246,10 +244,10 @@ Future<List<E>> findAllByAltText<E extends Element>(
   Node container,
   /*TextMatch*/ dynamic text, {
   bool exact = true,
-  NormalizerFn Function([NormalizerOptions]) normalizer,
-  Duration timeout,
-  Duration interval,
-  QueryTimeoutFn onTimeout,
+  NormalizerFn Function([NormalizerOptions?])? normalizer,
+  Duration? timeout,
+  Duration? interval,
```

Comment:
I'm on the fence here... should we make these two nullable e'where, or create two const singleton values that are used throughout as default arg values?

Leaning towards the second option since if they were actually ever set to null - I would think unexpected behavior would ensue....

Reply:
These do [get defaulted once then get passed to `waitFor`](https://github.com/Workiva/react_testing_library/blob/master/lib/src/dom/async/wait_for.dart#L61-L67). I do like the idea of making them non-nullable everywhere - I can do this for `interval` since it defaults to a constant, but how would I do it for `timeout` since it needs to call `getConfig`?

Reply:
> how would I do it for timeout since it needs to call getConfig?

Can a fn arg be made late?

Reply:
I don't think so 😞